diff.py

```diff
@@ -2578,8 +2588,26 @@ def process(dump: str, config: Config) -> List[Line]:
 
         if not row:
             continue
+        
+        # Check if the currrent line has "OFFSET <SYMBOL>:"
+        function_label_match = re.match(r"^[0-9a-f]+ <(.*)>:$", row)
+
+        if function_label_match:
+            function_name = function_label_match.groups()[0] + ":"
 
-        if re.match(r"^[0-9a-f]+ <.*>:$", row):
+            if config.diff_function_symbols:
+                # If diffing function symbols is enabled
+                # Add the symbol to the diff output
+                
+                output.append(
+                    Line(
+                        mnemonic=function_name, # this is needed for score changes
+                        diff_row=function_name, # this is needed to be set for highlighting
+                        original=function_name, # this is the value that is printed
```

Comment:
This is the wrong place for those comments. Ones in the `Line` class would be useful though, for sure, although they would require quite some care to get right so don't feel obliged